deploy/infra/aws/cloudformation/jump-server.json

```diff
@@ -0,0 +1,83 @@
+[
```

Comment:
do we need to store this file in code repo? I think it's related to specific deployment